messaging/src/test/java/org/axonframework/messaging/responsetypes/InstanceResponseTypeSerializationTest.java

```diff
@@ -26,22 +26,22 @@
 
 /**
  * Tests serialization capabilities of {@link InstanceResponseType}.
- * 
+ *
  * @author JohT
  */
 class InstanceResponseTypeSerializationTest extends AbstractResponseTypeTest<AbstractResponseTypeTest.QueryResponse> {
```

Comment:
I have already removed these in PR https://github.com/AxonFramework/AxonFramework/pull/3740 :-[
But, it is what it is. I will likely deal with the merge conflicts.